tasks/openjdk.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Add OpenJDK PPA
+  apt_repository: repo='ppa:openjdk-r/ppa'
+  when: "{{ ansible_distribution_version | version_compare('14.04', '=') }}"
```

Comment:
AFAICT from [this listing](https://launchpad.net/~openjdk-r/+archive/ubuntu/ppa),  `ppa:openjdk-r/ppa` contains packages for OpenJDK 7. Do you have any opposition to installing OpenJDK 7 from the PPA as well?